test/matchers/toBeWithinRange.test.js

```diff
@@ -0,0 +1,23 @@
+import * as matcher from 'src/matchers/toBeWithinRange';
+
+expect.extend(matcher);
+
+describe('.toBeWithinRange', () => {
+  test('passes when given array is within range', () => {
+    expect([4, 5, 7, 9]).toBeWithinRange(4, 9);
```

Comment:
I don't think an array should be passed here.

Reply:
Do you mean

```
const arrayList = [4, 5, 7, 9];
expect(arrayList).toBeWithinRange(4, 9);
```

Reply:
No, I thought the type of expected was a number, not an array, but now I see you have `actual.find()`.

I'm wondering if this should accept a number instead (or maybe in addition).

Reply:
> I'm wondering if this should accept a number instead

There's already an assertion for that called as `toBeWithin()`. So do we really need this `toBeWithinRange()` as well to accept a number?